etc/make-hilla-badge-url.sh

```diff
@@ -0,0 +1,29 @@
+
+echo
+echo
+echo
+echo
+
+echo "Green badge"
+echo "=========="
+echo "Suggested URL alias: quarkus-hilla-h-${hilla_version/./-}-g"
```

Comment:
why "hilla-h-..." ?

Reply:
> What is doing `/./-`

It is a string replacement: 2.4-SNAPSHOT --> 2-4-SNAPSHOT.
It is done only because tinyurl does not allow dots in the alias

Reply:
> why "hilla-h-..." ?

It stands for `quarkus-hilla` with `h`illa version x.y.x